lib/engine.js

```diff
@@ -79,7 +79,7 @@ module.exports = function container (get, set, clear) {
     }
 
     function isFiat () {
-      return !s.currency.match(/^BTC|ETH|XMR|USD|EUR|USDT$/)
+      return !s.currency.match(/^BTC|ETH|XMR|USDT$/)
```

Comment:
`USD|EUR` is not fiat? What is the thought process behind this?

Reply:
If we leave it, we get like that: ``` 0.00000 BTC  1000.00000 EUR```
If we return to @carlos8f [origin](https://github.com/carlos8f/zenbot/blob/6162ffdfb72f6542a871a34ae5e76cf8cfb90737/lib/engine.js#L64), we have correctly displayed currencies: ```0.00000 BTC  1000.00 EUR```